### ISSUE TITLE
not-translated:The arithmetic sequence a_ia
​i
​​ a,...

### ISSUE THREAD

**Issue by KhanBugz** [Khan/khan-i18n#104806]
Url: https://es.khanacademy.org/math/integral-calculus/sequences_series_approx_calc/calculus-series/e/arithmetic_series

User Agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36

Problem Text: 
The arithmetic sequence a_ia
​i
​​ a, start subscript, i, end subscript is defined by the formula:
a_1 = 4a
​1
​​ =4a, start subscript, 1, end subscript, equals, 4
a_i = a_{i - 1} + 11a
​i
​​ =a
​i−1
​​ +11a, start subscript, i, end subscript, equals, a, start subscript, i, minus, 1, end subscript, plus, 11
Find the sum of the first 650650650 terms in the sequence.

2322775
Hint #1
Getting started

Let's write out the first few terms of the series:
4 + 15 + 26 + 37...4+15+26+37...4, plus, 15, plus, 26, plus, 37, point, point, point
We're dealing with an arithmetic series because the difference between terms is constant. That is, each term is 111111 greater than the one before it.
We need a formula to compute the sum of the terms.
Hint #2
Formula for arithmetic series

The sum S_nS
​n
​​ S, start subscript, n, end subscript of a finite arithmetic series is
~~~~~S_n = \dfrac {\left(a_1 + a_n \right)}{2} \cdot n     S
​n
​​ =
​2
​
​(a
​1
​​ +a
​n
​​ )
​​ ⋅nspace, space, space, space, space, S, start subscript, n, end subscript, equals, start fraction, left parenthesis, a, start subscript, 1, end subscript, plus, a, start subscript, n, end subscript, right parenthesis, divided by, 2, end fraction, dot, n
where a_1a
​1
​​ a, start subscript, 1, end subscript is the first term, a_na
​n
​​ a, start subscript, n, end subscript is the last term, and nnn is the number of terms.
Hint #3
What do we need to use the formula?

The first term (a_1 = \greenD4)(a
​1
​​ =4)left parenthesis, a, start subscript, 1, end subscript, equals, start color greenD, 4, end color greenD, right parenthesis and the number of terms (n = \blueD{650})(n=650)left parenthesis, n, equals, start color blueD, 650, end color blueD, right parenthesis are given in the question.
We need to find the last term (a_n)(a
​n
​​ )left parenthesis, a, start subscript, n, end subscript, right parenthesis.
Hint #4
Step 1: Find a_na
​n
​​ a, start subscript, n, end subscript (the last term)

There are 650 -1= 649650−1=649650, minus, 1, equals, 649 terms after the first term. The sequence increases by 111111 for each new term. So, the sequence increases by a total of 649 \cdot 11 = 7139649⋅11=7139649, dot, 11, equals, 7139 from where it starts at 444. That means the last term must be 4+7139 = \goldD{7143}4+7139=71434, plus, 7139, equals, start color goldD, 7143, end color goldD.
In other words, a_n = \goldD{7143}a
​n
​​ =7143a, start subscript, n, end subscript, equals, start color goldD, 7143, end color goldD.

User Description: 


Next Steps: 
If this is a translation issue, fix it. Otherwise verify what is wrong, add a comment in English describing the issue and assign to tzjames.
